README.md

```diff
@@ -1,288 +1,19 @@
-<a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
 
-Modify this file to match your project and remove sections that don't apply.
+## Hello Microverse
 
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
 
-OPTIONAL SECTIONS:
-- FAQ
 
-After you're finished please remove all the comments and instructions!
--->
+Hello Microverse 
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
+I added  Html and css linters git \\ 
+Added a header in index.html and gave it some styling
 
-  <h3><b>Microverse README Template</b></h3>
+This project is about understanding linter, pull request and .gitignore
+Understanding how to write content (i.e commit messages, branch naming and summaries) about project in a professional way 
+Learn how to create new branch and merging them into the main branch after comparing 
```

Comment:
- I also noticed that here you are missing the  [Microverse Readme template](https://github.com/microverseinc/readme-template). This makes your project incomplete. Every section of the readme template is required with the exception of the FAQ section which is optional. So please make sure to add the README template to your project and fill in all the necessary contents :+1:

- [ ] Follow the steps I have listed below to customize it to suit this project:

    • Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it :100: 

    • For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. For the Client section, kindly include HTML and CSS :+1: 

    • For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. :wink: 

    • For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
        ◦ Installation of Git
        ◦ Installation of NPM
        ◦ A working browser
        ◦ Internet connection
        ◦ A code editor like Visual Studio Code
        ◦ Installation of Live Server plugging in Visual Studio Code
        ◦ Any other tool you can think about :100: 

    • For the Setup section, kindly add instructions for cloning your repository on a local machine :+1: 

    • For the Install section, kindly add instructions for setting up linters used in this project :+1: 

    • For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.

- For the Future Features section include 2 or more future features.

 - Kindly implement all the changes listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student.  
 
 - Lastly add your MIT license link :+1: